Base/HomagConnect.Base/Services/ServiceBase.cs

```diff
@@ -104,6 +104,32 @@ protected async Task PostObject(Uri uri)
             await response.EnsureSuccessStatusCodeWithDetailsAsync(request);
         }
 
+        protected async Task<T> PostObject<T>(Uri uri, T data)
+        {
+            var request = new HttpRequestMessage
+            {
+                Method = HttpMethod.Post,
+                RequestUri = uri,
+                Content = new StringContent(JsonConvert.SerializeObject(data, SerializerSettings.Default), Encoding.UTF8, "application/json")
+            };
+
+            var response = await Client.SendAsync(request).ConfigureAwait(false);
+            await response.EnsureSuccessStatusCodeWithDetailsAsync(request);
+
+            var result = await response.Content.ReadAsStringAsync();
+            var deserializeObject = JsonConvert.DeserializeObject<T>(result, SerializerSettings.Default);
+
+            return deserializeObject;
+        }
+
+        protected async Task PatchObject(Uri uri)
```

Comment:
For the update functionalities I have added the PatchObject method, which will hopefully work like I implemented it as .netstandard 2.0 does not support HttpMethod.Patch.

I would prefer a Patch over a Put when updating

Reply:
Do we also want to provide the updated Entity to the customer, after doing the update just as we do for the creation with the Post?

Reply:
yes